addons/mail/static/src/core/common/channel_member_model.js

```diff
@@ -20,10 +20,28 @@ export class ChannelMember extends Record {
     create_date;
     /** @type {number} */
     id;
+    isThreadDisplayed = Record.attr(false, {
+        compute() {
+            if (this.store.discuss.isActive && !this.store.env.services.ui.isSmall) {
+                return this.thread?.eq(this.store.discuss.thread);
+            }
+            return Boolean(this.store.ChatWindow.get({ thread: this.thread }));
+        },
+        onUpdate() {
```

Comment:
Let's imagine we switch from chat window to discuss:
- The component thread from the chat window is destroyed
- The component thread from discuss is not yet mounted
- The `isThreadDisplayed` field will change from true to false while it shouldn't, the rendering process make it disappears but it is semantically still displayed.
That why I figured it would be better to rely on state instead of rendering of components. We just have to enrich the compute when it will be needed.